scrapscript.py

```diff
@@ -394,8 +388,6 @@ def parse(tokens: typing.List[Token], p: float = 0) -> "Object":
         # we can match variants in MatchFunction
         # It needs to be higher than the precedence of the && operator so that
         # we can use #true() and #false() in boolean expressions
-        # It needs to be higher than the precedence of juxtaposition so that
```

Comment:
Please leave this comment; still applies

Reply:
Oh, I see. Thanks for pointing that out. I've restored this hunk.